lance/lance-common/src/test/java/org/apache/gravitino/lance/common/ops/gravitino/TestLanceDataTypeConverter.java

```diff
@@ -270,10 +275,11 @@ void testUnsupportedTypeThrowsException() {
     assertThrows(UnsupportedOperationException.class, () -> CONVERTER.fromGravitino(unparsedType));
   }
 
-  @Test
-  void testToGravitinoNotImplemented() {
-    assertThrows(
-        UnsupportedOperationException.class, () -> CONVERTER.toGravitino(ArrowType.Utf8.INSTANCE));
+  @ParameterizedTest(name = "[{index}] {0}")
+  @MethodSource("toGravitinoArguments")
+  void testToGravitino(String testName, Field arrowField, Type expectedGravitinoType) {
+    Type convertedType = CONVERTER.toGravitino(arrowField);
+    assertEquals(expectedGravitinoType, convertedType);
```

Comment:
Can you verify `arrowFileld == CONVERTER.fromGravitino(CONVERTER.toGravitino(arrowField))`？

Reply:
`CONVERTER.fromGravitino(Type type)` returns `ArrowType`, not an Arrow Field, so we cannot do the verify `arrowField == CONVERTER.fromGravitino`